services/skus/skus.go

```diff
@@ -144,16 +144,37 @@ func newOrderItemReqForSubID(set map[string]model.OrderItemRequestNew, subID str
 
 func skuNameByMobileName(subID string) (string, error) {
 	switch subID {
-	case "brave.leo.monthly", "beta.leo.monthly", "nightly.leo.monthly", "braveleo.monthly":
+	// Android Leo Monthly.
+	case "brave.leo.monthly", "beta.leo.monthly", "nightly.leo.monthly":
 		return "brave-leo-premium", nil
 
-	case "brave.leo.yearly", "beta.leo.yearly", "nightly.leo.yearly", "braveleo.yearly":
+	// iOS Leo Monthly.
+	case "braveleo.monthly", "beta.braveleo.monthly", "nightly.braveleo.monthly":
```

Comment:
question: I take it `ios` definitely doesn't have a `.` like the other types

Reply:
Yes. There is no a `.` in there for iOS.